packages/protocol/contracts/layer1/verifiers/IProverRegistry.sol

```diff
@@ -52,27 +51,20 @@ interface IProverRegistry {
     error INVALID_PRC10(bytes32 pcr10);
 
     event InstanceAdded(
-        uint256 indexed id,
-        address indexed instance,
-        address replaced,
-        uint256 validUntil
+        uint256 indexed id, address indexed instance, address replaced, uint256 validUntil
     );
     event VerifyProof(uint256 proofs);
 
     /// @notice register prover instance with quote
-    function register(
-        bytes calldata _report,
-        ReportData calldata _data
-    ) external;
+    function register(bytes calldata _report, ReportData calldata _data) external;
 
     /// TODO: should we need to add teeType?
```

Comment:
Could you go over the TODOs and remove those that aren't relevant anymore? There are a few more in other files.